### PR TITLE
Add has_scope bump in the CHANGELOG

### PR DESCRIPTION
**Pull request by rafaelfranca**
Also can we disable the protected branch in master? It is not good to not being able to do small commits like this directly.

### CODE REVIEW

File: CHANGELOG
Comment (tf) [inline CHANGELOG]: I think, it should be "now able", right?

Comment (deivid-rodriguez) [inline CHANGELOG]: Do you mean "are able"?

Comment (rafaelfranca) [reply CHANGELOG]: :+1:

**Comment by deivid-rodriguez** [comment]
@rafaelfranca I think the way @timoschilling configures this is to allow admins to override the setting. In any case, I find it nice so people get a chance to review even the small changes. In this case, CHANGELOG could've become more confusing for a while if this had landed straight into master.

**Comment by rafaelfranca** [comment]
My experience working in this project for more than 6 years and in OSS in general is that restricting contributors to do their work without peer review decrease the number of contributions if they are not motivated to do it. In my case I'm not motivated to work in this project but I'd occasionally help, but restricting what I can do will just make me avoid contributing.

I do agree it is nice to give people opportunity to review, but review can happen in different stages. I personally review after the fact to unblock others.

**Review by varyonic — APPROVED** [review]

**Comment by varyonic** [comment]
Sorry Rafael, it's just the way we ActiveAdmin maintainers have agreed to work together for now.  As David noted we believe you can override if needed.

**Comment by rafaelfranca** [comment]
I can't. I'm not an admin.

**Comment by rafaelfranca** [comment]
BTW, I'm still a contributor of this project. I did agree to move the project to activeadmin org, but I didn't agree to remove me as contributor neither to change the way the project works.

**Comment by timoschilling** [comment]
@rafaelfranca That automatically happens during the movement from jose to activeadmin. I have invited you now.

**Comment by rafaelfranca** [comment]
Thanks! Thank you for giving a new home to this project and helping with it.

**Comment by timoschilling** [comment]
Your welcome, we have to thank you for working on inherited_resources for rails 4 and rails 5 ❤️

**Pull request merged**
